### PR TITLE
docs: Group guides by categories

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Group different guides in categories. 

Selection_091

### CODE REVIEW

File: docs/guides/advise/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (eiffel-fl) [inline docs/guides/advise/_index.md:1]: Can you please reword the commit message to precise this is related to the website?
When I first saw your PR I was like: "Hum... Isn't it already done?"

Comment (mauriciovasquezbernal) [reply docs/guides/advise/_index.md]: I reworded it. Thanks!

Comment (eiffel-fl) [reply docs/guides/advise/_index.md]: One small nit: `s/caterogirization/categorization/`.

**Comment by mauriciovasquezbernal** [comment]
I repushed again because I pushed the wrong branch in the last try.

**Review by eiffel-fl — APPROVED** [review]
Let's merge it!

**Review by marga-kinvolk — APPROVED** [review]
LGTM

**Pull request merged**